audio/src/main/java/edu/artic/audio/AudioDetailsFragment.kt

```diff
@@ -119,6 +146,33 @@ class AudioDetailsFragment : BaseViewModelFragment<AudioDetailsViewModel>() {
     }
 
 
+    private fun bindTranslationSelector(viewModel: AudioDetailsViewModel) {
+        // TODO: Consider replacing 'Spinner' with a more capable class; possibly one from a 3-party library
+        val selectorView: Spinner = exo_translation_selector
+        viewModel.availableTranslations
+                .map {
+                    LanguageAdapter(selectorView.context, it)
```

Comment:
why rebind the `adapter` everytime? We should be setting the items list to the adapter only. No need to reset the adapter.

Reply:
I guess for this PR no need. I can submit a MR which fixes this

Reply:
Oh, good point. Since I'm using a subclass of `ArrayAdapter` to store the list we would need to go through that class's API (`::clear` and `::addAll`) instead of modifying the list directly. I should check whether JW wrote an extension for that.....